src/Corax/Queries/IMatchComparer.cs

```diff
@@ -3,12 +3,19 @@
 
 namespace Corax.Queries
 {
+    public enum SortingType
```

Comment:
why is this needed?

Reply:
This is a workaround for the current Corax API.

@redknightlois can you add `Alphanumeric` to `MatchCompareFieldType`? Because alphanumeric will always operate on strings, I think.

Reply:
Natural Order is only reasonable to work when you are doing roman language encodings (ASCII). I dont think we need to include anything else, this is kinda a replacement of the normal sorting only on certain occasions (should be Opt-in) but that is something that only RavenDB can know.